install/kubernetes/helm/istio/charts/gateways/templates/clusterrole.yaml

```diff
@@ -1,5 +1,5 @@
 {{- range $key, $spec := .Values }}
-{{- if and (ne $key "global") (ne $key "enabled") }}
+{{- if and (ne $key "enabled") }}
```

Comment:
same comment throughout PR.  Any idea if there is a specific reason we just aren't using requirements.yaml to solve this problem?